src/styles/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
I guess that you should make a variable for transition duration because you have that value more than once in your code.
And also would be good to add on hover effect cursor: pointer

Reply:
ok.